### PR TITLE
Clarify Documentation (Part 2)

### PR DESCRIPTION
**Pull request by Lange**
- [x] Mention that Electron's autoUpdater behavior must be implemented for auto updating to work. ([Example](https://github.com/develar/onshape-desktop-shell/blob/master/src/AppUpdater.ts))
- [x] More prominently link to the [Wiki](https://github.com/electron-userland/electron-builder/wiki), specifically [Publishing Artifacts](https://github.com/electron-userland/electron-builder/wiki/Publishing-Artifacts).
- [x] Add docs explaining that the OSX DMG installer window size is based on the size of `background.png`.
- [x] More clearly document the recommended draft/release workflow, as discussed in [this comment](https://github.com/electron-userland/electron-builder/issues/340#issuecomment-212426562) on Issue #340.

Suggestions of other areas to improve are welcome.

### CODE REVIEW

File: docs/Publishing Artifacts.md [hunk 1]
```diff
@@ -13,20 +13,29 @@ Currently, only GitHub Releases is supported.
...
  ```json
-"release": "build"
-```
- and if you run `npm run release`, release will be drafted (if doesn't exists) and artifacts published.
\ No newline at end of file
+ "release": "build"
+ ```
+ and if you run `npm run release`, a release will be drafted (if doesn't already exist) and artifacts published.
+
+ ## Recommended Workflow
```
Comment (develar) [inline docs/Publishing Artifacts.md]: Header without content :)

File: README.md [hunk 2]
```diff
@@ -83,8 +83,10 @@ Please note — packaged into an asar archive [by default](https://github.com/el
 * `.full-nupkg`: required for Squirrel.Windows.
 * `-amd64.deb` and `-i386.deb`: Linux Debian package. Please note — by default the most effective [xz](https://en.wikipedia.org/wiki/Xz) compression format used.
 
-You need to deploy somewhere releases/downloads server.
-Consider to use [Nuts](https://github.com/GitbookIO/nuts) (GitHub as a backend to store assets) or [Electron Release Server](https://github.com/ArekSredzki/electron-release-server).
+For auto updating to work, you must implement and configure Electron's [`autoUpdater`](http://electron.atom.io/docs/latest/api/auto-updater/) module ([example](https://github.com/develar/onshape-desktop-shell/blob/master/src/AppUpdater.ts)).
```
Comment (develar) [inline README.md]: I think, we should mention [electron-squirrel-startup](https://github.com/mongodb-js/electron-squirrel-startup) here (see https://github.com/electron/windows-installer#handling-squirrel-events) Or may be just add this link https://github.com/electron/windows-installer#handling-squirrel-events .

Comment (Lange) [inline README.md]: That link is already present a few lines above, in the bullet point for the `.exe` artifact. Should I move it, or leave it where it is?

Comment (develar) [inline README.md]: Sorry, misread, your change is ok, :+1:

**Pull request merged**